index.js

```diff
@@ -1,37 +1,42 @@
-var fs = require('fs');
+const fs = require('fs');
+const path = require('path');
 
 function WebpackTouch(options) {
-  var options = options || {}
-  this.filename = options.filename;
-  this.delay = options.delay? parseInt(options.delay) : 0;
-  this.ignoreOnError = options.ignoreOnError;
-  if (!this.filename) {
-    throw new Error('Require filename option')
-  }
-}
+  options = options || {};
+
+  return {
+    apply: apply,
```

Comment:
We should use:
this.apply = apply
then
new WebpackTouch({}) make more sense

Reply:
Ok, I'll bring back prototypes.